test/framework/src/main/java/org/elasticsearch/test/ESTestCase.java

```diff
@@ -674,6 +675,15 @@ public static DateTimeZone randomDateTimeZone() {
         return DateTimeZone.forID(randomFrom(ids));
     }
 
+    /**
+     * generate a random TimeZone from the ones available in java.time
+     */
+    public static TimeZone randomTimeZone() {
+        List<String> ids = Arrays.asList(TimeZone.getAvailableIDs());
```

Comment:
Could these be statically loaded so calls to this do not each need to do a sort?

Reply:
I'm just looking at how we currently do this for the joda DateTimeZone in `randomDateTimeZone`, I think that method would benefit slighty as well from making the sorted IDs static. Would make sense to maybe change this in one go?

Reply:
I switched them both to use statically initialized and sorted lists